activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeBPMNTimerIT.java

```diff
@@ -266,17 +265,18 @@ public void shouldExecuteProcessWithTimerStartExtension() {
         //then
         await().untilAsserted(() -> {
 
-            securityUtil.logInAs("user");
+            securityUtil.logInAs("admin");
+
+            Page<ProcessInstance> processInstancePage = processBaseRuntime.getProcessInstancesPageAsAdmin();
 
-            Page<ProcessInstance> processInstancePage = processBaseRuntime.getProcessInstancesPage();
             //then
             assertThat(processInstancePage).isNotNull();
             assertThat(processInstancePage.getContent()).isNotEmpty();
 
             ProcessInstance processInstance = processInstancePage.getContent().get(0);
             assertThat(processInstance.getProcessDefinitionKey()).isEqualTo(VARIABLE_MAPPING_PROCESS_START_TIME);
 
-            List<VariableInstance> variables = processBaseRuntime.getProcessVariablesByProcessId(processInstance.getId());
+            List<VariableInstance> variables = processBaseRuntime.getProcessVariablesByProcessIdAsAdmin(processInstance.getId());
```

Comment:
don't we have test coverage for this ?

Reply:
ProcessBaseRuntime is a test helper class